examples/real_data_example/README.md

```diff
@@ -0,0 +1,47 @@
+# Seismic Data Extractor
+
+A C++ tool for extracting and processing 3D seismic data slices from MDIO datasets. The tool will extract a slice of a larger MDIO dataset and output the result to a NumPy array called seismic_slice.npy.
+
+This tool demonstrates working with the Poseidon dataset, a public seismic dataset available on AWS Earth, and is provided under a CC BY 4.0 license.
+
+## Overview
+
+This tool allows users to extract specific slices from 3D seismic data stored in MDIO format, with options to specify ranges for inline, crossline, and depth dimensions. The extracted data can be saved in NumPy format for further analysis.
+
+## Prerequisites
+
+- C++ compiler with C++17 support
```

Comment:
```suggestion
- GCC 11 *or better* or Clang14 *or better*
```
We should be emphasizing the hard requirement for the compiler versions rather than C++ standard as older versions are not supported and appear to be causing confusion.

Reply:
done